admin/src/scenes/profil/index.js

```diff
@@ -70,33 +70,86 @@ export default () => {
                     <p>Données personnelles</p>
                   </BoxTitle>
                   <BoxContent direction="column">
-                    <Item required title="E-mail" values={values} name="email" handleChange={handleChange} errors={errors} touched={touched} />
-                    {user.role === ROLES.REFERENT_DEPARTMENT ? <Item title="Département" disabled values={values} name="department" handleChange={handleChange} /> : null}
-                    {user.role === ROLES.REFERENT_REGION ? <Item title="Région" disabled values={values} name="region" handleChange={handleChange} /> : null}
+                    <Item
+                      required
+                      title="E-mail"
+                      values={values}
+                      name="email"
+                      handleChange={handleChange}
+                      errors={errors}
+                      touched={touched}
+                      type={"text"}
+                      validate={(v) => required && !v && requiredMessage}
```

Comment:
`required` n'existe pas ici

```suggestion
                      validate={(v) => !v && requiredMessage}
```